frame.go

```diff
@@ -124,9 +124,10 @@ type Prompt struct {
 }
 
 type Field struct {
-	Name        string `json:"name,omitempty"`
-	Sensitive   *bool  `json:"sensitive,omitempty"`
-	Description string `json:"description,omitempty"`
+	Name        string   `json:"name,omitempty"`
+	Sensitive   *bool    `json:"sensitive,omitempty"`
+	Description string   `json:"description,omitempty"`
+	Options     []string `json:"options,omitempty"`
```

Comment:
I've tried to keep the SDKs well-tested. Please add a test case for this.

Reply:
absolutely